west.yml

```diff
@@ -139,7 +139,8 @@ manifest:
       revision: c39888ff74acf421eeff9a7514fa9b172c3373f7
     - name: trusted-firmware-m
       path: modules/tee/tfm
-      revision: 4544ab9fc4d5305ae5de97c73a9356ef0f342f7a
+      revision: 867d714099168481d8ca4ff9de9bfa7c27a0ec63
+      import: true
```

Comment:
We should not import from Zephyr modules from Zephyr manifest.
Please revert to the code you initially had when using Zephyr mcuboot, like you had here:
https://github.com/zephyrproject-rtos/zephyr/commit/b34978fc5ce2322a53fbb0888fc9fa614219d7f6

```
    - name: tfm-mcuboot
      path: modules/tee/tfm/mcuboot
      repo-path: mcuboot
      revision: 69344636bee1f18a6703dc5a8e7a0edfd8128d45
```